packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -187,11 +187,10 @@ export const FeedContainer = ({
       )}
     >
       {isV1Search && shouldUseFeedLayoutV1 && (
-        <img
-          className="absolute left-0 top-0 w-full -translate-y-1/2"
-          src={cloudinary.feed.bg.layoutV1}
-          alt="Gradient background"
-        />
+        <div className="absolute -top-24 left-1/2 flex -translate-x-1/2 flex-row justify-center">
```

Comment:
why do we use both `-top-24` and `-translate-x-1/2`?

Reply:
We don't display the full size, instead, we show partial.

The `-translate-x-1/2` is to make it on absolute centered position working with `left-1/2`.

Reply:
OK, makes sense. I would maybe prefer to not mix absolute positioning with CSS transforms, but whatever works :)